modules/mbedtls/SCsub

```diff
@@ -8,84 +8,85 @@ env_mbed_tls = env_modules.Clone()
 # Thirdparty source files
 thirdparty_dir = "#thirdparty/mbedtls/library/"
```

Comment:
That could go in the `if` too :P (but it's not important)

Reply:
Indeed! Done XD